### PR TITLE
feat[WEB-8]: add CSS variables for fonts, colors, and screen sizes

### PR DESCRIPTION
**Pull request by GarnaA**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -1,4 +1,34 @@
 :root {
+  --screen-mobile-s: 320px;
+  --screen-mobile-m: 375px;
+  --screen-mobile-l: 425px;
+  --screen-tablet: 768px;
+  --screen-laptop: 1024px;
+  --laptop-l: 1440px;
```
Comment (AndrZakhara) [inline src/index.scss:7]: use the same naming for variables 
ex:
--screen-laptop
--screen-l
--screen-xl

File: src/index.scss [hunk 2]
```diff
@@ -1,4 +1,34 @@
...
+  --laptop-4k: 2560px;
+
+  --common-font: 'SST W20 Roman', sans-serif;
+  --common-font-bold: 'SST W20 Bold', sans-serif;
+  --common-font-en: 'SST W20 Roman', sans-serif;
+  --common-font-en-bold: 'SST W20 Bold', sans-serif;
```
Comment (AndrZakhara) [inline src/index.scss:13]: use only en fonts, remove W55.  names should be: SSTW20-Bold, SSTW20-Roman. check this pr and font naming: https://github.com/weblodz/folio/pull/3

**Review by AndrZakhara — COMMENTED** [review]
also remove dublicated vars from _variables.scss

**Review by MyHKo — COMMENTED** [review]
Agree with the previous reviewer

**Pull request closed without merging**